dev/core/test/com/google/gwt/dev/resource/impl/ResourceAccumulatorTest.java

```diff
@@ -25,13 +25,17 @@
 import java.util.Collections;
 import java.util.Comparator;
 import java.util.List;
+import java.util.Locale;
 import java.util.Set;
 
 /**
  * Tests for ResourceAccumulator.
  */
 public class ResourceAccumulatorTest extends TestCase {
 
+  private final static boolean IS_WINDOWS = System.getProperty("os.name")
```

Comment:
Do we have commons-lang3 in the classpath? We could use `SystemUtils.IS_OS_WINDOWS` then.